roles/reboot-host/tasks/main.yml

```diff
@@ -0,0 +1,12 @@
+---
+
+- name: "Reboot the host - {{ inventory_hostname }}"
+  shell: sleep 5 && shutdown -r now "Ansible Reboot of host"
```

Comment:
You can simply use the reboot module: https://docs.ansible.com/ansible/latest/modules/reboot_module.html ansible <=2.6 is EOL and it's not used anymore, right?

Reply:
That's actually a very good suggestion. Let me update the PR.